bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/ResumingStreamingResultScanner.java

```diff
@@ -143,6 +150,7 @@ private void handleScanTimeout(ScanTimeoutException rte) throws IOException {
       reissueRequest();
     }
     else {
+      rpcMetrics.incrementFailureCount();
```

Comment:
It might be interesting to distinguish between unretryable errors and retries exhausted...